packages/edit-site/src/components/create-template-part-modal/index.js

```diff
@@ -39,17 +39,21 @@ import {
 } from '../../utils/template-part-create';
 
 export default function CreateTemplatePartModal( {
-	closeModal,
+	area: defaultArea = TEMPLATE_PART_AREA_DEFAULT_CATEGORY,
```

Comment:
Oops I mean to name the prop itself `defaultArea` instead of aliasing it within the component.

Reply:
Ah ok, thanks for clearing that up, I'll fix it up before merging.